File: 1816-truncate-sentence/1816-truncate-sentence.js
/**
 * @param {string} s
 * @param {number} k
 * @return {string}
 */
var truncateSentence = function (s, k) {
    let ss = s.split(" ");
    let ret = [];
    for (let i = 0; i < k; i++) {
        ret.push(ss[i]);
    }
    return ret.join(" ");
};

File: 1816-truncate-sentence/1816-truncate-sentence.java
class Solution {
    public String truncateSentence(String s, int k) {
        String[] ss = s.split(" ");
        String[] ret = new String[k];
        for (int i = 0; i < k; i++) {
            ret[i] = ss[i];
        }
        return String.join(" ", ret);
    }
}